recipe/meta.yaml

```
@@ -54,7 +47,7 @@ test:
     - conda-libmamba-solver  # [osx-arm64]
```

```suggestion
    - conda-libmamba-solver
```

Should be added everywhere, just in case